client/webpack/dev.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = merge(common, {
 		proxy: {
 			"/api": "http://localhost:3100",
 		},
+		static: false,
```

Comment:
Hi Doris, No sure what this line is doing? can you explaine please?

Reply:
I didn't create this change of file manually. Yeah, I think Saleh explained it, so do I need to do anything about this?